Makefile

```diff
@@ -4,6 +4,17 @@ IMG ?= quay.io/${USER}/odh-model-controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.26
 
+ENGINE ?= docker
+#####################################
+# Podman only parameter
+ARCH ?= x86_64
+# Default for docker, no value
+ARCH_PARAMETER =
+ifeq ($(ENGINE),podman)
+    ARCH_PARAMETER = --arch $(ARCH)
```

Comment:
Would `--platform` work? Such arg seems to be supported both by docker and podman, so we no longer would need conditionals.

Reply:
work if you enforce os to linux as well.
--arch does it automatically.

--platform
```
Error: creating build container: choosing an image from manifest list docker://registry.access.redhat.com/ubi8/go-toolset:1.19: no image found in manifest list for architecture amd64, variant "", OS darwin

```